deep_quoridor/src/train.py

```diff
@@ -135,32 +72,35 @@ def train_dqn(
         default=True,
         help="Disable Weights & Biases logging",
     )
+    parser.add_argument(
+        "-p",
+        "--players",
+        nargs="+",
+        type=player_with_params,
+        help=f"List of players to compete against each other. Can include parameters in parentheses. Allowed types {AgentRegistry.names()}",
+    )
```

Comment:
Maybe it's worth adding also an argument for the renderers, since I see you manually switched to pygame?